release.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -xe
+rm -rf dist/
+mkdir dist
+
+mvn clean install -DskipTests -P slack.notifier
```

Comment:
Can we change this to `mvn clean package` instead of `mvn clean install`? Also we don't use profiles in this project so `-P slack.notifier` is not needed right or am I missing something?